src/dispatch/cli.py

```diff
@@ -520,13 +523,23 @@ def dispatch_scheduler():
     """Container for all dispatch scheduler commands."""
     # we need scheduled tasks to be imported
     from .document.scheduled import sync_document_terms  # noqa
-    from .feedback.scheduled import feedback_daily_digest  # noqa
-    from .incident.scheduled import daily_report, auto_tagger  # noqa
+    from .feedback.scheduled import daily_report  # noqa
+    from .incident.scheduled import daily_report, auto_tagger, close_incident_reminder  # noqa
+    from .incident_cost.scheduled import calculate_incidents_opportunity_cost  # noqa
     from .report.scheduled import incident_report_reminders  # noqa
     from .tag.scheduled import sync_tags  # noqa
-    from .task.scheduled import sync_tasks, create_task_reminders  # noqa
+    from .task.scheduled import (
+        create_task_reminders,
+        daily_sync_task,
+        sync_active_stable_tasks,
+        sync_tasks,
+    )  # noqa
     from .term.scheduled import sync_terms  # noqa
-    from .workflow.scheduled import sync_workflows, sync_active_stable_workflows  # noqa
+    from .workflow.scheduled import (
+        sync_workflows,
+        daily_sync_workflow,
+        sync_active_stable_workflows,
+    )  # noqa
```

Comment:
@kevgliss I added some imports that were missing (e.g. `close_incident_reminder`). Some of the functions we're importing (e.g. `sync_workflows`, `sync_tasks`) do not have the `scheduler` and `background_task` decorators. Mind checking that we're importing only the ones we need?

Reply:
By the looks of it, the functions you mentioned are called by other functions that have those decorators.

Reply:
Right, so we should only import the ones that have the decorators correct?

Reply:
Yes, only the ones we decorators need to be imported.